to_production.txt

```diff
@@ -1,31 +1,51 @@
 # New
 ofl/calsans # https://github.com/google/fonts/pull/9232
-ofl/cascadiacode # https://github.com/google/fonts/pull/9226
-ofl/cascadiamono # https://github.com/google/fonts/pull/9227
 ofl/comicrelief # https://github.com/google/fonts/pull/9075
 ofl/coralpixels # https://github.com/google/fonts/pull/9274
+ofl/exile # https://github.com/google/fonts/pull/9281
+ofl/kanchenjunga # https://github.com/google/fonts/pull/8431
+ofl/playpensansarabic # https://github.com/google/fonts/pull/9188
+ofl/playpensansdeva # https://github.com/google/fonts/pull/9304
+ofl/playpensanshebrew # https://github.com/google/fonts/pull/9187
+ofl/playpensansthai # https://github.com/google/fonts/pull/9198
 ofl/tagesschrift # https://github.com/google/fonts/pull/8735
 
 # Upgrade
+# Deleted: apache/robotomono # https://github.com/google/fonts/pull/9282
```

Comment:
What does this mean? What did we delete from Roboto Mono?

Reply:
I think that this means that we removed it from the apache directory as part of the Apache to OFL license conversion.

Reply:
this is the change from apache to OFL, right @m4rc1e?